frontend/src/bundles/common/helpers/get-last-workout/get-last-workout.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { type WorkoutResponseDto } from '~/bundles/workouts/types/types.js';
+
+const getLastWorkout = (
+    workouts: WorkoutResponseDto[],
+): WorkoutResponseDto | null => {
+    if (!workouts || workouts.length === 0) {
+        return null;
+    }
+    const sortedWorkouts = [...workouts].sort((a, b) => {
+        return (
+            new Date(b.workoutStartedAt).getTime() -
+            new Date(a.workoutStartedAt).getTime()
+        );
+    });
+
+    const lastWorkout = sortedWorkouts[0];
+    if (!lastWorkout) {
+        return null;
+    }
```

Comment:
why do we need this check? We already asserted that there is !== 0 workouts.
I would suggest to change a little
0 workouts - return null
1 workout - return this workout immediately
in other case sort